src/others/helpers/map-utils.ts

```diff
@@ -50,3 +27,34 @@ export const mapAidRequestsToFeatures = (mapData: RequestMapDataPoint[], locatio
   });
 };
 
+export type RegionMetadata = { [id: string]: {amount: number, description: string } };
+type RegionData = {
+  region_id: string;
+  city_name: string;
+  amount: number;
+};
+
+export const mapRegionIdsToMetadata = (mapDataPoints: MapDataPoint[], translateLocation: (city_id: number) => Location | undefined): RegionMetadata => {
```

Comment:
**issue / naming**

If you decide to stick to the naming schema already in place, this would be called `groupByRegions`

Reply:
done